composer.json

```diff
@@ -28,7 +28,7 @@
         "phpstan/phpstan-phpunit": "^0.12",
         "phpunit/phpunit": "^8.0",
         "friendsofphp/php-cs-fixer": "^2.14",
-        "jms/serializer": "^1.13 || ^2"
+        "jms/serializer": "^2 || ^3"
```

Comment:
could we keep allowing 1.13 or is there a problem with that version?

Reply:
No, it's not possible. 1.13 does not support PHP 8.x and was dropped from the `liip/metadata-parser`.